### DISCUSSION TITLE
Change aggregation period in statistics-graph

### DISCUSSION THREAD

**Discussion by erik3rik** [home-assistant/frontend#9847] [Dashboard]
Currently the aggregation period is hard-coded to 1h.
In some cases, especially when doing really long-term measurement it would be interesting to see other options such as "daily" aggregation instead.

For example; If i have 6 months of data for outdoor temperature; i would like to have MIN/AVG/MAX on a daily basis not hourly. That way I would get a much more sensible graph compared to the extremely zig-zag graph where one hour corresponds to less than a pixel in width.

Same for energy consumption in the long run; I most likely don't care about hourly usage but the daily usage on a;, lets say, yearly basis. Sum of energy used per day is much more useful for that purpose, but when investigating daily patterns of course an hourly aggregation would be good. That is why I propose having that as an option; like

`aggregation: hourly` and `aggregation: daily` etc. in the graph (of if that is not possible for reasons like how the data is actually aggregated / stored, then at least make that option available on the sensor-level)

**Comment by nsmits** [comment]
I would really like this. The statistics-data is already present and perhaps the code can be re-used from the Energy-dashboard (2019.9 version where week/month/year-aggregation is built in). This feature is also requested in #9719. In #9734 the same question is raised; how can the long term statistics be utilized (in other card types).
